common.py

```diff
@@ -44,9 +44,6 @@
     "bytecode.cud",
 ]
 
-s3 = boto3.resource("s3")
-s3_client = boto3.client("s3")
-
```

Comment:
Tests won't run if boto code is run during an import step. I struggled with pulling this out because it meant some bigger changes to the code and I tried to minimize any code changes in this PR.